dss-server/src/main/java/io/github/ztkmkoo/dss/server/DssServer.java

```diff
@@ -38,4 +39,6 @@
      * @return self(for method chain)
      */
     DssServer<S> addDssService(S service);
+
+    DssServer<S> addExceptionHandler(DssExceptionHandler exceptionHandler);
```

Comment:
Could you like to add the document of this method?

Reply:
Ok. I will add document.